content/en/docs/tasks/tools/install-kubectl-windows.md

```diff
@@ -26,11 +26,12 @@ The following methods exist for installing kubectl on Windows:
 
 1. Download the [latest release {{< param "fullversion" >}}](https://dl.k8s.io/release/{{< param "fullversion" >}}/bin/windows/amd64/kubectl.exe).
 
-   Or if you have `curl` installed, use this command:
+   use this command:
 
    ```powershell
-   curl -LO "https://dl.k8s.io/release/{{< param "fullversion" >}}/bin/windows/amd64/kubectl.exe"
+   Invoke-WebRequest -Uri "https://dl.k8s.io/release/{{< param "fullversion" >}}/bin/windows/amd64/kubectl.exe" -OutFile "kubectl.exe"
    ```
+  
```

Comment:
```suggestion
   curl.exe -LO "https://dl.k8s.io/release/{{< param "fullversion" >}}/bin/windows/amd64/kubectl.exe"
```

Reply:
I think Invoke-Webrequest would be more conventient for native windows users since powershell is the goto command line for most power  users.